dts/arm/nordic/nrf51822.dtsi

```diff
@@ -133,6 +134,18 @@
 			label = "SPI_1";
 		};
 
+		/* nRF TEMP Peripheral:
+		 * .. Default enabled,
+		 * .. Interrupt priority:
+		 * .. .. Range: 0-1, Default: 1
+		 */
+		temp: temp@4000C000 {
+			compatible = "nordic,nrf-temp";
+			reg = <0x4000C000 0x1000>;
+			interrupts = <12 1>;
+			status = "ok";
+		};
```

Comment:
Shouldn't we provide a label property for the node, like for all others?

Reply:
Currently, the label property for this node is not used for anything - so i did not see any reasons to add it. This is one of the questions I have been wondering about, should a driver label/name be defined in Kconfig - or should it be define in DTS? 

The driver name for this TEMP driver is currently CONFIG_TEMP_NRF5_NAME, ref: https://github.com/zephyrproject-rtos/zephyr/pull/12034/files#diff-4adad1dc7c1f3fd67902b233f21c7ef2R128. For me, it makes sense that a SW drivers name is defined in Kconfig because that is where a user will enable the driver - and thus at the time they enable it, they need to know the name of the driver so that they can use it in their code. If the name is defined in DTS, he will have to search through the DTS file to find the name after enabling it through Kconfig - that seems like more of a hassle? 

I can add a label just to align it with the other nodes, but it wont be used for anything.

Reply:
Added label property for TEMP node.